saito-core/src/core/consensus/peer.rs

```diff
@@ -39,10 +42,17 @@ pub struct Peer {
     pub last_msg_at: Timestamp,
     pub wallet_version: Version,
     pub core_version: Version,
+    pub key_list_rate_limiter: RateLimiter,
+    pub handshake_rate_limiter: RateLimiter,
 }
 
 impl Peer {
     pub fn new(peer_index: u64) -> Peer {
+        let mut key_list_rate_limiter = RateLimiter::default();
+        key_list_rate_limiter.set_limit(RATE_LIMITS[0]);
```

Comment:
Ok. I was suggesting you don't need any arrays since you only need 2 limiters. no need to use those definitions. (remove RATE_LIMITS and directly mention the values here). 

Anyway what I suggested was, if we are using an array, to keep the indices defined. i.e : HANDSHAKE_LIMIT = 0, etc...